docs/manual/inventory.md

```diff
@@ -46,6 +46,41 @@ Lab nodes are grouped under their kinds in the inventory so that the users can s
               ansible_host: <mgmt-ipv4-address>
     ```
 
+## Removing `ansible_host` var
+If you want to use a plugin[^1] that doesn't play well with the `ansible_host` variable injected by containerlab in the inventory file, you can leverage the `ansible-no-host-var` label. The label can be set on per-node, kind, or default levels; if set, containerlab will not generate the `ansible_host` variable in the inventory for the nodes with that label.  
+Note that without the `ansible_host` variable, the connection plugin will use the `inventory_hostname` and resolve the name accordingly if network reachability is needed.
+
+=== "topology file"
+    ```yaml
+    name: ansible
+      topology:
+        defaults:
+          labels:
+            ansible-no-host-var: "true"
+        nodes:
+          node1:
+          node2:
+    ```
+=== "generated ansible inventory"
+    ```yaml
+    all:
+      children:
+        linux:
+          hosts:
+            clab-ansible-node1:
+            clab-ansible-node2:
+    ```
+=== "ansible docker host inventory file"
+    ``` yaml
+    all:
+      children:
+        linux:
+          hosts:
+            clab-ansible-linux-host:
+              ansible_host: <mgmt-ipv4-address>
+              ansible_docker_host: clab-ansible-linux-host
+    ```
```

Comment:
@tobbbles what does this example show? Can't quite get my head around it ;)

Reply:
Ah, that's leftover from the previous implementation that I completely forgot to remove 🤦 Cleaned this up now :)